File: modules/react-native-mlkit-core/android/src/main/java/red/infinite/reactnativemlkit/core/RNMLKitLabel.kt
package red.infinite.reactnativemlkit.core

import expo.modules.kotlin.records.Field
import expo.modules.kotlin.records.Record

data class RNMLKitLabel(
    @Field var text: String,
    @Field var confidence: Float,
    @Field var index: Int? = null,
) : Record

File: modules/react-native-mlkit-core/ios/RNMLKitLabel.swift
import Foundation
import ExpoModulesCore

public struct RNMLKitLabel:Record {
    @Field
    var text: String = ""
    @Field
    var index: Int = 0
    @Field
    var confidence:CGFloat=0
    
    public init(text: String, index: Int, confidence: CGFloat) {
        self.text = text
        self.index = index
        self.confidence = confidence
    }
    
    public init() {}
}
